src/pages/Home.vue

```diff
@@ -20,12 +20,35 @@
             </div>
         </div>
 
-        <div class="container card card-body py-4">
+        <div class="container card card-body py-4 my-4">
             <h4>What is the Mappers' Guild?</h4>
             <div class="text-secondary">
                 <a href="https://osu.ppy.sh/wiki/Community/Mappers_Guild" target="_blank">The Mappers' Guild</a> is a hub for beatmaps associated with <a href="https://osu.ppy.sh/beatmaps/artists" target="_blank">osu!'s Featured Artists</a>!
             </div>
             <hr />
+            <div>For the most newest info about Featured Artist mapping, check out the channels below on the <a href="https://discord.gg/ppy" target="_blank">osu!dev Discord server</a>.</div>
```

Comment:
```suggestion
            <div>For the latest info about Featured Artist mapping, check out the channels below on the <a href="https://discord.gg/ppy" target="_blank">osu!dev Discord server</a>.</div>
```
this gives me "the bestest free-to-win rhythm game" vibes, anyway, "most newest" is more words than necessary and newest alone should suffice, however i feel using latest is better phrasing for this

Reply:
this was from me rewording the sentence oops